modules/services/hound.nix

```diff
@@ -0,0 +1,87 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.hound;
+
+  configFile = pkgs.writeText "hound-config.json" (
+    builtins.toJSON {
+      inherit (cfg) max-concurrent-indexers dbpath repos;
+      health-check-url = "/healthz";
+    });
+
+  houndOptions = [
+    "--addr ${cfg.addr}"
+    "--conf ${configFile}"
+  ];
+
+  execStart = pkgs.writeScript "houndd-execstart" ''
+    #!${pkgs.runtimeShell}
+    export PATH="${pkgs.mercurial}/bin:${pkgs.git}/bin''${PATH:+:}$PATH"
+    exec ${cfg.package}/bin/houndd ${concatStringsSep " " houndOptions}
+  '';
+
+in
+
+{
+  meta.maintainers = [ maintainers.adisbladis ];
+
+  options.services.hound = {
+    enable = mkEnableOption "hound";
+
+    package = mkOption {
```

Comment:
Is it likely that anybody would like to use a different package for this service? Only `pkgs.hound` seems to exist in Nixpkgs so it would seem natural to me that one would use an overlay if one wanted a different package.

Reply:
Not really.. It was for my benefit when writing the module.